examples/tv-casting-app/linux/main.cpp

```diff
@@ -114,6 +115,30 @@ HelpOptions helpOptions("tv-casting-app", "Usage: tv-casting-app [options]", "1.
 
 OptionSet * allOptions[] = { &cmdLineOptions, &helpOptions, nullptr };
 
+static void MyBoundDeviceChangedHandler(const EmberBindingTableEntry & binding, chip::DeviceProxy * peer_device, void * context)
+{
+    using namespace chip;
```

Comment:
using namespace chip is already at the top level. Do we need these using statements?

Reply:
removed